src/backend/apps/utils/middleware.py

```diff
@@ -0,0 +1,23 @@
+from django.middleware.common import CommonMiddleware
+from django.conf import settings
+from django.urls import is_valid_path
+
+
+class CustomCommonMiddleware(CommonMiddleware):
+    def should_redirect_with_slash(self, request):
+        """
+        Return True if settings.APPEND_SLASH is True and appending a slash to
+        the request path turns an invalid path into a valid one.
+
+        Exclude our catch-all template view when deciding if a path matches our urlconf.
+        """
+        if settings.APPEND_SLASH and not request.path_info.endswith("/"):
+            urlconf = getattr(request, "urlconf", None)
+            if urlconf is None:
+                urlconf = 'urls.base'
+            if not is_valid_path(request.path_info, urlconf):
+                match = is_valid_path("%s/" % request.path_info, urlconf)
```

Comment:
Also maybe add some logs in here just in case this behavior is hit and causes weirdness? Maybe a smidge easier to debug

Reply:
Maybe with a low logger level, so they aren't always displaying? Will be a lot of logs otherwise.

Reply:
Ah maybe I'm misunderstanding, I was assuming this was only hit when we accidentally forget a slash?

Reply:
I'm not sure how often this will actually get hit. Anytime there is no slash, it will get get. I believe that's any request made to load the nuxt resource, for example:
![Screenshot 2023-01-21 at 8 36 34 PM](https://user-images.githubusercontent.com/21694127/213900845-954a06f6-8d6b-48bb-aa97-7a43877acf8b.png)